docs/README.md

```diff
@@ -0,0 +1,22 @@
+Simpleflow documentation
+========================
+
+This directory hosts simpleflow documentation.
+
+
+Installing
+----------
+
+    pip install pip-tools
+    ./script/pip-sync
+
+
+Running for development
+-----------------------
+
+    ./script/run
+
+Deploying
+---------
+
+    ./script/deploy
```

Comment:
EDIT: readthedocs deployment is not that easy given that I don't have permissions on the original project. I'll figure out what to do, but for now I'll mark the docs as being moved to Github.